freqtrade/tests/testdata/download_backtest_data.py

```diff
@@ -10,14 +10,14 @@
 
 parser = misc.common_args_parser('download utility')
 parser.add_argument(
-        '-p', '--pair',
-        help='JSON file containing pairs to download',
-        dest='pair',
-        default=None
+    '-p', '--pair',
+    help='JSON file containing pairs to download',
+    dest='pair',
+    default=None
 )
 args = parser.parse_args(sys.argv[1:])
 
-TICKER_INTERVALS = [1, 5]  # ticker interval in minutes (currently implemented: 1 and 5)
+TICKER_INTERVALS = ['1m', '5m']  # ticker interval in minutes (currently implemented: 1 and 5)
```

Comment:
comment is no longer valid

Reply:
Fixed https://github.com/gcarq/freqtrade/pull/600/commits/65c5a0b308490fae1dda5217cf822b51415e0de1